Setup/MakePortableArchive.cmd

```diff
@@ -125,6 +125,485 @@ xcopy /y ..\Plugins\Statistics\GitImpact\bin\%Configuration%\GitImpact.dll GitEx
 IF ERRORLEVEL 1 EXIT /B 1
 xcopy /y ..\Plugins\Statistics\GitStatistics\bin\%Configuration%\GitStatistics.dll GitExtensions\Plugins\
 IF ERRORLEVEL 1 EXIT /B 1
+xcopy /y ..\GitExtensions\bin\%Configuration%\Plugins\TfsIntegration.dll GitExtensions\Plugins\
+IF ERRORLEVEL 1 EXIT /B 1 ) 
```

Comment:
❗️ A bunch of stray `)` characters on these lines

Reply:
It is WIP.  I am making sure portable is a FULL release and not missing anything.

Reply:
Sorry. the ) was confusing.  Kind of skipped right by it.  

I'll mod my script.  I bet I know what happened.  Thanks.